tests/raftstore/test_split_region.rs

```diff
@@ -302,4 +302,65 @@ fn test_node_split_overlap_snapshot() {
 fn test_server_split_overlap_snapshot() {
     let mut cluster = new_server_cluster(0, 3);
     test_split_overlap_snapshot(&mut cluster);
-}
\ No newline at end of file
+}
+
+fn test_apply_new_version_snapshot<T: Simulator>(cluster: &mut Cluster<T>) {
+    // truncate the log quickly so that we can force sending snapshot.
+    cluster.cfg.store_cfg.raft_log_gc_tick_interval = 20;
+    cluster.cfg.store_cfg.raft_log_gc_limit = 2;
+
+    // We use three nodes 1, 2, 3 for this test.
+    cluster.bootstrap_region().expect("");
+    cluster.start();
```

Comment:
Can we merge those two function into a single function ? I saw the pattern many times.